.travis.yml

```diff
@@ -2,7 +2,6 @@ dist: trusty
 language: python
 python:
 - '2.7'
-- '3.3'
```

Comment:
How come we remove python 3.3?

Reply:
Travis was failing for that version suddenlyÑ https://travis-ci.org/socialpoint-labs/sheetfu/builds/594308093.

The logs are saying: "google-api-python-client requires python3 version >= 3.4". I'm assuming this is a new dependency issue.